README.md

```diff
@@ -4,9 +4,31 @@
 
 Build software only once using [Nix](https://nixos.org/nix/) with the help of [Cachix](https://cachix.org).
 
+This action will configure Cachix and invoke `nix-build`.
+
+## Why do I need this
+
+Because you'd like for your CI to be fast. Let me explain.
+
+Caching on a typical CI doesn't work in favor of Nix.
```

Comment:
```suggestion
Directory-based caching on a typical CI doesn't work for Nix.
```

Reply:
That's a strong statement, it should then say "work well".